Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterCreateEditWidget.cpp

```diff
@@ -130,14 +121,38 @@ namespace EMStudio
         }
     }
 
-    void ParameterCreateEditDialog::Init()
+    void ParameterCreateEditWidget::Init(const EMotionFX::Parameter* editParameter)
```

Comment:
I recommend just having one init function if Init() and Reinit() does the same thing. Normally in GUI class my idea for Init function is to initialize everything that is needed, including memory allocations, verses on Reinit we reset the values without major memory allocations.